google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -36,6 +36,9 @@ dependencies {
     compile files(project(':google-cloud-tools-plugin').sourceSets.main.output)
     compile files(project(':google-cloud-tools-plugin:jps-plugin').sourceSets.main.output)
     compile files("${System.properties['java.home']}/../lib/tools.jar")
+    compile ('com.google.cloud.tools:appengine-plugins-core:0.1.4') {
```

Comment:
Since this is also defined in the google-cloud-tools-plugin module maybe we should extract out the version to a variable so we don't have to update it in two places when upgrading the lib.